app/controllers/v0/account_controls_controller.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+module V0
+  class AccountControlsController < SignIn::ServiceAccountApplicationController
+    service_tag 'identity'
+    VALID_CSP_TYPES = %w[logingov idme dslogon mhv].freeze
+
+    def credential_index
+      if @service_account_access_token.user_attributes['icn'].blank?
+        raise SignIn::Errors::MissingParamsError.new message: 'icn is not defined'
+      end
+
+      serialized_user_verifications = serialize_user_verifications(user_verifications: fetch_verifications_by_icn)
+      Rails.logger.info('[V0::AccountControlsController] credential_index',
+                        { icn: params[:icn], requested_by: @service_account_access_token.user_identifier })
```

Comment:
hi Gia, Im the lead engineer for the VA.gov Identity team. Unfortunately without higher level changes to how users are identified within our systems we must continue to log ICN. The value is too entrenched into our performance monitoring tools and if we were forced to remove it with a VA wide identifier to take its place it would impact our ability to ensure Veterans services on VA.gov are not being impacted negatively. 

CC: @tblackwe